_includes/Formulas/scope-amq.md

```diff
@@ -0,0 +1,21 @@
+#### ActiveMQ Request Step Scope
+
+ActiveMQ Request steps add the step execution values described in the example JSON below to the formula context. The formula context is then passed from step-to-step, allowing you to use these values in any subsequent steps in your formula.
+
+```json
+{
+  "my-step-name": {
+    "request": {
+        "body": "{\"message\":\"This is a test message.\"}",
+        "url": "amqp://otqaqsml:tPpXwTl7-iMtezRmyJmD-y2U_XbroYpW@jaguar.rmq.cloudamqp.com/otqaqsml",
+        "exchange": "main",
+        "queue": "myqueue"
+    }
+  }
+}
+
+```
+Example references to ActiveMQ Request scope:
+
+* `$steps.my-step-name.request`
```

Comment:
Two things:
1) I think this should be wrapped in curly brackets again, `${...}`
2) I don't believe JS variables can have dashes in them and be referenced like that.  Have you tried that in a formula?  I believe you'd need to do `steps['my-step-name'].request` to reference that object.  I would recommend naming your steps without dashes so you can reference them in that dot notation.